quantum/rgblight.c

```diff
@@ -120,16 +124,10 @@ void setrgb(uint8_t r, uint8_t g, uint8_t b, LED_TYPE *led1) {
 
 
 uint32_t eeconfig_read_rgblight(void) {
-  #ifdef __AVR__
-    return eeprom_read_dword(EECONFIG_RGBLIGHT);
-  #else
-    return 0;
-  #endif
+  return eeprom_read_dword(EECONFIG_RGBLIGHT);
```

Comment:
Is this working on ATSAM too?

Reply:
Hmm, not AFAIK. I think maybe I should keep the #define but expand it for stm32

Reply:
__AVR__ for ... well, yeah, but STM32 uses QMK_STM32:
https://github.com/qmk/qmk_firmware/blob/3cf179be61a10860b2b66aecf2ec5ca6f0e30605/build_keyboard.mk#L140-L144

As far as I can tell, `MCU_FAMILY` only exists on ChibiOS based ARM boards. ATSAM uses something different, entirely. 

(otherwise, we could just use __arm__)

Reply:
I take this back.  `STM32_EEPROM_ENABLE` seems to be a better option here, as that's what is in the core eeconfig.c file, and should be used instead